services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -193,6 +205,7 @@ public int hashCode() {
         private Boolean consistentRead;
         private Expression filterExpression;
         private List<String> attributesToProject;
```

Comment:
Can we remove this and convert attribute name strings as they are added? If we don't think could cause unexpected and surprising behavior when they overwrite existing attributes